crates/language/src/language.rs

```diff
@@ -2460,4 +2514,59 @@ mod tests {
             "LSP completion items with duplicate label and detail, should omit the detail"
         );
     }
+
+    #[test]
```

Comment:
Thanks for testing this. I believe it’s safe to remove this now that we’ve confirmed it works. I don’t want to add an additional dependency, and this configuration is unlikely to change frequently.

Reply:
I think we should keep the test, but we can move the dependency to a dev dependency: https://github.com/zed-industries/zed/pull/34861#discussion_r2225449983

Reply:
I kept it and updated it to test the new and old formats/names, and moved the dep to a dev dep.